pkg/server/server_test.go

```diff
@@ -401,6 +402,21 @@ func TestServerLoaduserDatabase(t *testing.T) {
 	}
 }
 
+func TestServerLoadUserDatabases(t *testing.T) {
+	copier := fs.NewStandardCopier()
+	require.NoError(t, copier.CopyDir("../../test/data_v1.0.1", "data_v1.0.1"))
```

Comment:
Wouldn't it be better to have a separate dataset just for this failure case?
Simething like `test/data_v1.0.1_dbsettings_test`

Reply:
it could be, but actually its the migration needed for any db up to v1.0.1.